dns/google/cloud/dns/client.py

```diff
@@ -115,3 +104,18 @@ def zone(self, name, dns_name=None, description=None):
         """
         return ManagedZone(name, dns_name, client=self,
                            description=description)
+
+
+def _item_to_zone(iterator, resource):
```

Comment:
(no action needed)

I was thinking that you could use a lambda of functools.partial to avoid having to separately define a function, but it's probably too clever.